deploy/helm/kubecf/assets/operations/instance_groups/brain-tests.yaml

```diff
@@ -0,0 +1,48 @@
+- path: /releases/-
+  type: replace
+  value:
+    name: brain-tests
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "brain-tests") }}
+    version: v0.0.5
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "brain-tests") }}
+
+- path: /instance_groups/-
+  type: replace
+  value:
+    azs:
+    - z1
+    env:
+      bosh:
+        agent:
+          settings:
+            serviceAccountName: {{ .Values.deployment_name }}-tests-brain
```

Comment:
I don't think that's a thing anymore (IIRC, you want `.Release.Name`).

Reply:
Will make the change to release name. Beyond that, that is what @f0rmiga told me to use.

Reply:
Done with commit b4a6f34cf652a6e983167b8319a88217acada94f

Reply:
We replaced all `.Values.deployment_name` to `.Release.Name`. `deployment_name` doesn't exist anymore.

Reply:
Yes. Already resolved.